src/go/k8s/tests/e2e/create-topic-given-issuer-with-client-auth/01-assert.yaml

```diff
@@ -40,3 +40,28 @@ status:
     - reason: Ready
       status: "True"
       type: Ready
+
+
+---
+
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: schema-registry-api-client
```

Comment:
huh? the name of the cert does not contain the name of the cluster? 🤔 how will that work in cluster with multiple RPs